client/app/bundles/course/duplication/components/CourseDropdownMenu.tsx

```diff
@@ -73,7 +73,7 @@ const CourseDropdownMenu: FC<CourseDropdownMenuProps> = (props) => {
             }
           >
             <MyLocation
-              className={`${currentCourseId === selectedCourseId && 'bg-blue-300'}`}
+              className={`${currentCourseId === selectedCourseId && 'fill-blue-500'}`}
```

Comment:
Why `fill-blue-500`? Isn't our convention to use `bg-`?

Reply:
This component (MyLocation) is defining an icon, and that coloring is for that icon only.
Though at first I tried to make the color consistent with what we used so far (`bg-`) but the color generated was quite different than what it used to be, and this `fill-blue-500` is the one that's closest to what we used to set for this component

Reply:
In that case, then put the color on the IconButton itself
```<IconButton color={`${currentCourseId === selectedCourseId ? 'info' : 'default'}`}```

Other examples:  
https://github.com/Coursemology/coursemology2/blob/021438e497269c6aeb547d54274161efb2724db7/client/app/bundles/course/forum/components/buttons/ReplyButton.tsx#L12  
https://github.com/Coursemology/coursemology2/blob/021438e497269c6aeb547d54274161efb2724db7/client/app/bundles/course/lesson-plan/pages/LessonPlanShow/MilestoneAdminTools.jsx#L94

Reply:
roger that!